### DISCUSSION TITLE
<title> There is no commit, but the record for that year remains.

### DISCUSSION THREAD

**Discussion by seongjin77** [community/community#43705] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![img](https://user-images.githubusercontent.com/96804958/211274692-c1d592b3-0d91-4e5b-971c-f60951dd1b4d.png)

I joined in 2021. I want to delete the year record that I don't need. What should I do?

**Comment by jongwooo** [comment]
@seongjin77 You can create related ticket on [GitHub Support](https://support.github.com/request) portal.

**Reply by jongwooo** [reply]
같은 계정명을 예전에 사용했던 사람이 있었을 수도 있어요. GitHub Support에 문의하시면 조치받으실 수 있을 것 같습니다!

**Reply by seongjin77** [reply]
답변 감사합니다!
